RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1017,7 +1051,8 @@ namespace {
         const double absX = std::abs(x);
         if (absX > maxX)
           return false;
-        const double absIndex = std::floor(std::abs(absX / size - 0.5));
+        //const double absIndex = std::floor(std::abs(absX / size - 0.5));
+        const double absIndex = std::floor(absX / size + 0.5);
```

Comment:
Indeed, this a known bug. Unfortunately, it was discovered during the integration when DeepTau v2 training was already completed. If we want to merge this PR to the central CMSSW, in order to keep compatibility with DeepTau v2 training, we'll need to have an external flag that indicates whatever to apply the "old-bugged" definition of the cell or the corrected one.